<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDRdaemonFECGui</class>
 <widget class="QWidget" name="SDRdaemonFECGui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>261</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>372</width>
    <height>261</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>SDRdaemon</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_freq">
     <item>
      <layout class="QVBoxLayout" name="deviceUILayout">
       <item>
        <layout class="QHBoxLayout" name="deviceButtonsLayout">
         <item>
          <widget class="ButtonSwitch" name="startStop">
           <property name="toolTip">
            <string>start/stop acquisition</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrbase/resources/res.qrc">
             <normaloff>:/play.png</normaloff>
             <normalon>:/stop.png</normalon>:/play.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="ButtonSwitch" name="record">
           <property name="toolTip">
            <string>Toggle record I/Q samples from device</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../../../sdrbase/resources/res.qrc">
             <normaloff>:/record_off.png</normaloff>:/record_off.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="deviceRateLayout">
         <item>
          <widget class="QLabel" name="deviceRateText">
           <property name="toolTip">
            <string>I/Q sample rate kS/s</string>
           </property>
           <property name="text">
            <string>00000k</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="freqLeftSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="ValueDial" name="centerFrequency" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>16</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Monospace</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>SizeVerCursor</cursorShape>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="toolTip">
        <string>Record center frequency in kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="freqUnits">
       <property name="text">
        <string> kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="freqRightlSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_address">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="gridLayout_corr">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Auto Corr</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="dcOffset">
       <property name="toolTip">
        <string>DC Offset auto correction</string>
       </property>
       <property name="text">
        <string>DC</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="ButtonSwitch" name="iqImbalance">
       <property name="toolTip">
        <string>IQ Imbalance auto correction</string>
       </property>
       <property name="text">
        <string>IQ</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="absTimeText">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Stream timestamp</string>
       </property>
       <property name="text">
        <string>2015-01-01  00:00:00.000</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bufferGaugesLayout">
     <item>
      <widget class="QProgressBar" name="bufferGaugeNegative">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Main buffer read/write positions unbalance: write leads read lags</string>
       </property>
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
       <property name="invertedAppearance">
        <bool>true</bool>
       </property>
       <property name="format">
        <string>%v</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bufferGaugeCenterDot">
       <property name="maximumSize">
        <size>
         <width>2</width>
         <height>10</height>
        </size>
       </property>
       <property name="text">
        <string>.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="bufferGaugePositive">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>10</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Main buffer read/write positions unbalance: read leads write lags</string>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_freq_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="streamLayout">
     <item>
      <widget class="QToolButton" name="framesComplete">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Frames complete</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../sdrbase/resources/res.qrc">
         <normaloff>:/locked.png</normaloff>:/locked.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="sampleRateText">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Actual sample rate (kS/s)</string>
       </property>
       <property name="text">
        <string>00000.000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineStream1">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bufferRWBalanceText">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Main buffer read/write positions unbalance (%): positive means read leads</string>
       </property>
       <property name="text">
        <string>-00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineStream2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="curNbBlocksText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Total number of blocks retrieved per frame</string>
       </property>
       <property name="text">
        <string>128</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineStream3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="avgNbBlocksText">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Average number of blocks retrieved per frame</string>
       </property>
       <property name="text">
        <string>000.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineStream4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="curNbRecoveryText">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Number of recovery blocks used per frame</string>
       </property>
       <property name="text">
        <string>000</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineStream5">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="avgNbRecoveryText">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Average number of recovery blocks used per frame</string>
       </property>
       <property name="text">
        <string>000.0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="lineStream6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="bufferLenSecsText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>28</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Buffer length in seconds</string>
       </property>
       <property name="text">
        <string>00.0</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_rateTime">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="addressLayout">
     <item>
      <widget class="QLabel" name="addressLabel">
       <property name="text">
        <string>Addr:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="address">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Local data connection IP address</string>
       </property>
       <property name="text">
        <string>000.000.000.000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dataPortLabel">
       <property name="text">
        <string>D:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="dataPort">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Local data connection port</string>
       </property>
       <property name="text">
        <string>00000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="controlPortLabel">
       <property name="text">
        <string>C:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="controlPort">
       <property name="toolTip">
        <string>Remote control port</string>
       </property>
       <property name="text">
        <string>00000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="iaddressSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line_freq">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout1">
     <item>
      <widget class="QLabel" name="freqLabel">
       <property name="text">
        <string>Fc:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="freq">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Set center frequency (kHz)</string>
       </property>
       <property name="text">
        <string>00000000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="freqUnitLabel">
       <property name="text">
        <string>kHz</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="decimLabel">
       <property name="text">
        <string>Dec:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="decim">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Decimation</string>
       </property>
       <property name="currentIndex">
        <number>3</number>
       </property>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>32</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>64</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fcPosLabel">
       <property name="text">
        <string> Fp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fcPos">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Center frequency position (Infradyne, Supradyne, Centered)</string>
       </property>
       <property name="currentIndex">
        <number>2</number>
       </property>
       <item>
        <property name="text">
         <string>Inf</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Sup</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cen</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sendButton">
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Send commands to remote SDRdaemon instance</string>
       </property>
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="controlLayout2">
     <item>
      <widget class="QLabel" name="sampleRateLabel">
       <property name="text">
        <string>SR:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="sampleRate">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Sample rate (kS/s)</string>
       </property>
       <property name="text">
        <string>00000</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="specificParmsLAbel">
       <property name="text">
        <string>k Sp:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="specificParms">
       <property name="toolTip">
        <string>Other parameters that are hardware specific</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="padLayout">
     <item>
      <spacer name="verticalPadSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ValueDial</class>
   <extends>QWidget</extends>
   <header>gui/valuedial.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ButtonSwitch</class>
   <extends>QToolButton</extends>
   <header>gui/buttonswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../sdrbase/resources/res.qrc"/>
 </resources>
 <connections/>
</ui>
